Tests/TuistCoreTests/Graph/Mappers/GraphMapperTests.swift

```diff
@@ -30,16 +30,16 @@ final class SequentialGraphMapperTests: TuistUnitTestCase {
         let input = Graph.test(name: "0")
         let first = AnyGraphMapper(mapper: { graph in
             XCTAssertEqual(graph.name, "0")
-            return Graph.test(name: "1")
+            return (Graph.test(name: "1"), [])
         })
         let second = AnyGraphMapper(mapper: { graph in
             XCTAssertEqual(graph.name, "1")
-            return Graph.test(name: "2")
+            return (Graph.test(name: "2"), [])
         })
         let subject = SequentialGraphMapper([first, second])
 
         // When
-        let got = try subject.map(graph: input)
+        let (got, _) = try subject.map(graph: input)
 
```

Comment:
it's worth adding another test for the `SequentialGraphMapper` to ensure side effects from different mappers are aggregated (possibly in the correct order too).

Reply:
That makes total sense. I'll add it.